### ISSUE TITLE
Wrong icon for Longhorn app in Apps & Marketplace

### ISSUE THREAD

**Issue by khushboo-rancher** [rancher/charts#1058]
The icon of Longhorn is appearing wrong on the App & Marketplace with master-head `2dca0c6`
<img width="1513" alt="Screen Shot 2021-03-18 at 7 32 52 PM" src="https://user-images.githubusercontent.com/60111667/111639351-2f12ed80-8821-11eb-96a2-25fe30d60ffd.png">

Expected Icon:
<img width="385" alt="Screen Shot 2021-03-18 at 7 40 24 PM" src="https://user-images.githubusercontent.com/60111667/111640039-ced07b80-8821-11eb-81e4-e19fb31ba47c.png">

**Comment by vincent99** [comment]
The book is the generic lack-of-icon that is shown by default. If there is an icon link then it's loaded asynchronously and replaced if it succeeds.

This chart has one defined and does load if you just follow the link.  But when passed through the rancher server icon link/proxy, it's getting served as content-type: text/plain and therefore the browser refuses to render it (because it says it's not an image).

It's probably something to do with the origin being github user content instead of our own CDN, even though they send the proper `image/svg+xml`

**Comment by innobead** [comment]
cc @jenting

**Comment by jenting** [comment]
Change the icon from .svg to .png fixes this issue.

However, the rancher server serves it as `content-type: text/plain` rather than `content-type: image/svg+xml`, it looks like an issue related to how the rancher server process it.

Moreover, the icon is wrong only in Apps & Marketplace view but is correct in Cluster Applications > Longhorn.
![image](https://user-images.githubusercontent.com/49380831/114817920-7030fa00-9ded-11eb-8dae-d97c653d3cd2.png)

**Comment by vincent99** [comment]
That's not fixing the problem; it's fixing the symptom by hacking around it and leaving the problem unfixed for when the next chart hits it.

All of our 1st party icons should be SVG, and the catalog endpoint should know how to serve them properly.

**Comment by innobead** [comment]
> That's not fixing the problem; it's fixing the symptom by hacking around it and leaving the problem unfixed for when the next chart hits it.
> 
> All of our 1st party icons should be SVG, and the catalog endpoint should know how to serve them properly.

Agree, this is why @jenting commented here, we still need to figure out the root cause. Who can help with this?
